examples/multiple-daemons/run.rs

```diff
@@ -34,9 +34,13 @@ async fn main() -> eyre::Result<()> {
     build_dataflow(dataflow).await?;
 
     let (coordinator_events_tx, coordinator_events_rx) = mpsc::channel(1);
-    let (coordinator_port, coordinator) =
-        dora_coordinator::start(None, ReceiverStream::new(coordinator_events_rx)).await?;
-    let coordinator_addr = SocketAddr::new(Ipv4Addr::LOCALHOST.into(), coordinator_port);
+    let coordinator_bind = SocketAddr::new(
+        IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)),
+        DORA_COORDINATOR_PORT_DEFAULT,
+    );
+    let (coordinator_addr, coordinator) =
+        dora_coordinator::start(coordinator_bind, ReceiverStream::new(coordinator_events_rx))
+            .await?;
     let daemon_a = run_daemon(coordinator_addr.to_string(), "A".into());
     let daemon_b = run_daemon(coordinator_addr.to_string(), "B".into());
```

Comment:
Similarly, could be good to set up a daemon in local address and one with a remote address, to cover the inter_daemon communication

Reply:
I opened https://github.com/dora-rs/dora/issues/475 to track this idea.